### ISSUE TITLE
Resize performance really slow

### ISSUE THREAD

**Issue by sa-** [MachatoApp/machato#101]
When you try to resize the app horizontally the performance is really slow, can this be fixed?

**Comment by FliiFe** [comment]
You're right that this is a recurring problem in Machato. Despite many efforts, I haven't been able to fix those performance issues.

If anyone can help, I'd greatly appreciate it!
